### ISSUE TITLE
Fine-tuning based on the DETR architecture code, but the verification indicators are all 0

### ISSUE THREAD

**Issue by Flyooofly** [amirbar/DETReg#69]
Thanks for your work. I noticed that you open-sourced the detreg of the DETR architecture, and then I tried to use the pre-trained model on the imagenet dataset you provided to fine-tune training for my custom dataset. But I found that all the indicators are still 0 after more than fifty batches of pre-training. I have followed the tips in the related issues of DETR (https://github.com/facebookresearch/detr/issues?page=1&q=zero) , the num_calss was modified. Many people mentioned that DETR requires a large amount of training data, or fine-tuning. But I am currently using fine-tuning, and the number of fine-tuning datasets is about one thousand. But the effect is still very poor, may I ask why. It's normal for me to use deformable-detr architecture.
![image](https://user-images.githubusercontent.com/31236750/202667218-f009b79e-217e-4d32-8b78-13e15b268200.png)

**Comment by luoyq6** [comment]
Excuse me, hello. Why do I get the error shown in the picture when I use the command to evaluate? Could you give me some tips?
![b39205ed6c3af39c07dc62491331797](https://user-images.githubusercontent.com/110521846/218691652-ba97a034-69e7-4ead-a79a-b85146555eda.png)

**Comment by amirbar** [comment]
Apologies for the delay in response. What is the size of your dataset? For example, ImageNet is 1m images, so 50 epochs might be enough to train over it. If your dataset is significantly slower, you would need to increase the number of epochs proportionally. Also, make sure you don't drop the learning rate early (this should be mostly relevant for Deformable-DETR).
